### DISCUSSION TITLE
Awesome project

### DISCUSSION THREAD

**Discussion by Eccomi21** [cryinkfly/Autodesk-Fusion-360-for-Linux#217] [General]
You've done an outstanding job here, thank you so much. Works like a charm on my arch install.

**Comment by snrkl** [comment]
Hey - had some teething troubles with the exe dissapearing (#215)  but managed to wrangle it..

Just wanted to say @cryinkfly you've done an amazing thing here, and I solidly salute you.. There were two key apps that I am struggling to leave behind on the Mac - you have taken care of one of them! 

Thank you so much!
